app/src/test/java/org/simple/clinic/drugs/PrescriptionRepositoryTest.kt

```diff
@@ -75,4 +77,15 @@ class PrescriptionRepositoryTest {
       verify(dao).save(argThat { isEmpty() })
     }
   }
+
+  @Test
+  fun `when soft deleting a prescription, update the timestamp as well as set the sync status to pending`() {
+
+    val prescriptionId = UUID.randomUUID()
+
+    repository.softDeletePrescription(prescriptionId).blockingAwait()
+
+    verify(dao).softDelete(eq(prescriptionId), eq(true), any())
```

Comment:
Holy shit I did not know you can combine `eq` and `any()` in `verify`. I was always forced to pass in actual values or manually compare the values.